copilot-chat-request.el

```diff
@@ -333,7 +332,7 @@ Optional argument QUIET suppresses user messages when non-nil."
     (request url
       :type "GET"
       :headers headers
-      :parser 'json-read
+      :parser (apply-partially 'json-parse-buffer :object-type 'alist :false-object :json-false :null-object nil)
```

Comment:
`json.c` does not use `:json-false`, but `:false` is assigned by default, so there is no need to force a custom setting.